### DISCUSSION TITLE
Job stays in the `queued` state for several hours and cannot be canceled

### DISCUSSION THREAD

**Discussion by vyazelenko** [community/community#26225] [Actions]
<p>There is build job (<a href="https://github.com/real-logic/aeron/actions/runs/37488855" rel="nofollow noopener">https://github.com/real-logic/aeron/actions/runs/37488855</a>) in the <a href="https://github.com/real-logic/aeron" rel="nofollow noopener">https://github.com/real-logic/aeron</a> repository that has status <code>queued</code> for several hours and any attempt to cancel this job have no effect. Other jobs on the same repo have completed since this one was stuck.</p>

**Comment by weide-zhou** [comment]
<p>Hi <a class="mention" href="/vyazelenko">@vyazelenko</a> ,</p>
<p>I confirmed the check runs in the link are all completed(pass or fail), and verified via restapi, the status is also ‘queued’ indeed.</p>
<p>According to the policy, please ask <a href="https://support.github.com/contact" rel="nofollow noopener">private support</a> to help resolve it, thanks.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/2X/3/36d309cd4ae513276dce3c7dd7163e535ee3f440.png" data-download-href="/uploads/short-url/7OZVIcfuANZ19YlabhkP8jooaly.png?dl=1" title="Queuestatus.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/52772761/181096149-82164eaf-9ffe-4a20-b912-ad3421d2617c.png" alt="Queuestatus.png" data-base62-sha1="7OZVIcfuANZ19YlabhkP8jooaly" width="585" height="500" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/36d309cd4ae513276dce3c7dd7163e535ee3f440_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">Queuestatus.png</span><span class="informations">607×518 19.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by vyazelenko** [comment, marked as answer]
<p>Thanks, I’ve created a support ticket.</p>

**Comment by aristide-jou** [comment]
![Capture](https://github.com/community/community/assets/125312768/2eb4c13b-0c87-473b-8e85-8dcbba1c0ded)

**Reply by aristide-jou** [reply]
please how to stop this job
